client/webpack.config.node.js

```diff
@@ -40,6 +43,16 @@ const fileLoader = FileConfig.loader( {
 
 const commitSha = process.env.hasOwnProperty( 'COMMIT_SHA' ) ? process.env.COMMIT_SHA : '(unknown)';
 
+function getMonorepoPackages() {
+	// find package.json files in all 1st level subdirectories of packages/
+	return globby.sync( 'packages/*/package.json' ).map( ( pkgJsonPath ) => {
+		// read the package.json file
+		const pkgJson = JSON.parse( fs.readFileSync( pkgJsonPath ) );
+		// create a package name regexp that matches all requests that start with it
+		return new RegExp( `^${ pkgJson.name }(/|$)` );
```

Comment:
`wepback-node-externals` also accepts plain strings, no need for a regexp:

> Can accept exact strings ('module_name'), regex patterns (/^module_name/), or a function that accepts the module name and returns whether it should be included.

https://www.npmjs.com/package/webpack-node-externals#optionsallowlist-

Reply:
The plain string option won't catch imports from package subdirectories, e.g., `import from 'i18n-calypso/react'`. The import source string needs to match exactly.

I'm not sure if any monorepo package actually uses this, but I went with the regexp that matches the subdirectories, too, to be safe.

Reply:
Yep, you are right. They use the module name to match the directory in node_modules, but the request to match items in the allowList

https://github.com/liady/webpack-node-externals/blob/master/index.js#L76